martin-core/Cargo.toml

```
@@ -12,7 +12,7 @@ rust-version.workspace = true
 homepage.workspace = true
 
 [features]
-default = ["fonts"]
+default = ["fonts", "sprites"]
```

this was a mistake in #2106 

```suggestion
default = ["fonts", "styles", "sprites"]
```